index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PORTFOLIO</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  </head>
+  <body>
+    <header class="hero">
+      <nav>
+        <a href="#" class="logo">Welcome</a>
+        <ul class = "nav-list">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <label for="check" class="hambuger-btn">
+          <i class = "fas fa-bars"></i>
+        </label>
+      </nav>
+      <div class="content">
+        <h1>Hey there. I'm Esther.</h1>
+        <h2>I'm a software developer</h2>
+        <P class = "supporting-text">I can help you build a feature, product or a website.
+        Look through some of my work and experience!
+        If you like what you see and have a project you need coded, dont
+          hesitate to contact me.
+        </p>
+      <div class="socialmedia">
+        <a href="https://medium.com/@mumoesther2021">
+            <img src="./images/socialmedia icons/mail.png" alt="Mail icon" />
+        </a>
+        <a href="https://github.com/mumo-esther">
+            <img src="./images/socialmedia icons/github.png" alt="github icon">
+        </a>
+        <a href="https://www.linkedin.com/in/esther-mawioo-58b636225/">
+            <img src="./images/socialmedia icons/linkedin.png" alt="linkedin icon">
+        </a>
+        <a href="https://twitter.com/EstherMawioo">
+            <img src="./images/socialmedia icons/twitter.png" alt="twitter icon">
+        </a>
+    </div>
+      <div class="disabled">
+        <img
+          src="./images/Disabled.png"
+          alt="disabled vector"
+        />
+      </div>
+    </div>
+  </header>
```

Comment:
- [ ] As suggested by the previous reviewer you have to wrap only toolbar section with `header` tag not the whole content, so Please adjust it.